static/templates/narrow_to_compose_recipients_tooltip.hbs

```diff
@@ -1,4 +1,7 @@
-<span>{{t 'Go to conversation' }}</span> <span class="hotkey-hint">({{shortcut_html}})</span>
-{{#if display_current_view}}
-<p class="narrow_to_compose_recipient_current_view_help">{{display_current_view}}</p>
-{{/if}}
+<div>
+    <span>{{t 'Go to conversation' }}</span>
+    {{#if display_current_view}}
+    <p class="narrow_to_compose_recipient_current_view_help tooltip-inner-content italic">{{display_current_view}}</p>
+    {{/if}}
+</div>
+{{hotkey_hints "Ctrl" "."}}
```

Comment:
How does this one look in the case that `display_current_view` is true? I worry a bit that the `hotkey_hints`  field should be in a different place in the HTML.

Reply:
As mentioned in my latest comment, as long as the hotkey hints are after the main content of the tooltip, they are rendered accurately. Posted screenshots to verify the same.